### PR TITLE
Morse Code Decipher Ruby Solution

### PR DESCRIPTION
**Pull request by Kanyorok**
- Create a method to decode a Morse code character.
- Create a method to decode an entire word in Morse code.
- Create a method to decode the entire message in Morse code.

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @Kanyorok and @maths29,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly remove the following header links in your readme as they are broken and have no linked sections: live demo, deployment, run tests

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: mosecode.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+def decode(morse_code)
+  morse_code = morse_code.strip # Remove leading and trailing spaces
+  morse_code_words = morse_code.split('   ') # Split the Morse code into words
+
+  morse_code_words.map do |word|
+    word.split(' ').map do |char|
+      morse_decipher(char)
+    end.join('')
+  end.join(' ')
+end
```
Comment (Johnadibe) [inline mosecode.rb]: > - [ ] Good job so far @Kanyorok and @maths29.👍 Your code for decoding Morse code is looking good overall. However, there's a slight adjustment needed to ensure it aligns perfectly with the project requirements. The project requirements state that you should create a decode_word method that decodes an entire word in Morse code, where every character in a word should be separated by a single space. Currently, your code only has 2 methods, a method to decode a Morse code character and a method to decode the entire message in Morse code whereby every word will be separated by 3 spaces. Kindly implement that👍
> 
> de
> 
> You could use this pseudocode below👇
> 
> ```
> def decode_word(word_morse)
>   # Split the input word_morse using a single space (' ')
>   # Use morse_decipher to decode each character
>   # Join the decoded characters without any spaces and return the result in uppercase
> end
> ```
>

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @Kanyorok and @maths29👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You used GitFlow correctly 👍🏽
✅ Linter checks passed✔️
✅ Created a method to decode a Morse code character that takes a string parameter, and returns the corresponding character in uppercase✔️
✅ Created a method to decode the entire message in Morse code that takes a string parameter, and returns the string representation in which every word is separated by 3 spaces✔️


## Required Changes ♻️

> - [ ] As this was missed by the previous reviewer, kindly fix your MIT link as it is broken.👍
> 
> mit

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @Kanyorok , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
Great effort in making the requested changes :+1: 
Code working fine :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**